### PR TITLE
Enable user to get bitcoin address from Ledger

### PR DESCRIPTION
**Pull request by ccali11**
After a deep dive into the ledger ecosystem and noting some limitations, going to default to using the `hw-app-btc` library for now, and not going to implement a sendTransaction method just yet. 

Moving on to #252 and going to check off of this branch so I can incorporate multiple tokens in the logic as I began to do with this PR.

### CODE REVIEW

File: apps/web/src/composables/wallet.ts [hunk 1]
```diff
@@ -53,7 +53,11 @@ export default function useWallet() {
     selectedAccount.value = address
   }
 
-  async function connectWallet(provider: ProviderString) {
+  const setSelectedToken = (token: TokenString) => {
+    selectedToken.value = token
+  }
+
+  async function connectWallet(provider: ProviderString, token?: TokenString) {
```
Comment (shanejearley) [inline apps/web/src/composables/wallet.ts:60]: @ccali11 what if we turn this into an options object (if you have time a ConnectWalletOptions.ts interface) and change 'token' to 'currency'?

File: apps/web/src/interfaces/MessageInit.ts [hunk 2]
```diff
@@ -3,4 +3,5 @@ import { ProviderString } from '@/types/ProviderString'
 export interface MessageInit {
     message: string;
     providerString: ProviderString;
+    token?: 'ETH' | 'BTC'
```
Comment (shanejearley) [inline apps/web/src/interfaces/MessageInit.ts:6]: @ccali11 Same let's find/replace all 'token' keys used in this sense to 'currency'.

**Review by shanejearley — CHANGES_REQUESTED** [review]

File: apps/web/src/composables/wallet.ts (hunk 1 above)
Comment (ccali11) [reply apps/web/src/composables/wallet.ts:60]: You got it!

File: apps/web/src/interfaces/MessageInit.ts (hunk 2 above)
Comment (ccali11) [reply apps/web/src/interfaces/MessageInit.ts:6]: 💯

**Pull request merged**